### DISCUSSION TITLE
Add styled-system to gitignore

### DISCUSSION THREAD

**Discussion by princemuel** [chakra-ui/panda#1031] [Q&A] [answered]
Hello, please I want to know if the styled-system directory should be ignored in git. 

It's always being regenerated and I'd like to not have to add all the files again each time I make a commit

**Comment by anubra266** [comment, marked as answer]
In the docs

![CleanShot 2023-07-14 at 21 26 04@2x](https://github.com/chakra-ui/panda/assets/30869823/c2abf3a4-7bc7-4e00-80a8-6acdec58d6c2)
